HoloPipelines/pipelines/pipelineDicom2glb.py

```diff
@@ -0,0 +1,27 @@
+from components import compDicom2numpy
+from components import compNumpy2obj
+from components import compObj2glbWrapper
+import pathlib
+import sys
+
+
+def main(dicomFolderPath, outputGlbPath, threshold):
+    generatedNumpyList = compDicom2numpy.main(str(pathlib.Path(dicomFolderPath)))
+    generatedObjPath = compNumpy2obj.main(
+        generatedNumpyList,
+        threshold,
+        str(
+            pathlib.Path.cwd()
+            .joinpath("output", "OBJ")
+            .joinpath(str(pathlib.PurePath(dicomFolderPath).parts[-1]))
+        )
+        + ".obj",
+    )
+    generatedGlbPath = compObj2glbWrapper.main(
```

Comment:
local variable 'generatedGlbPath' is assigned to but never used.

Print the path to find the new output?

Reply:
output path variable now used in final print of a pipeline